src/deepsparse/image_classification/schemas.py

```diff
@@ -53,8 +55,48 @@ def from_files(cls, files: List[str], **kwargs) -> "ImageClassificationInput":
             )
         return cls(images=files)
 
+    @staticmethod
+    def create_test_inputs(
+        batch_size: int = 1,
+    ) -> Dict[str, Union[Union[List[str], str, List[Any]]]]:
+        """
+        Create and return a test input for this schema
+
+        :param batch_size: The batch_size of inputs to return
+        :return: A dict representing inputs for Image Classification pipeline
+        """
+
+        sample_image_path = Path(__file__).parents[0] / "sample_images" / "basilica.jpg"
+        sample_image_abs_path = str(sample_image_path.absolute())
+
+        images = [sample_image_abs_path for _ in range(batch_size)]
+        return {"images": images}
+
+    def split(self) -> Generator["ImageClassificationInput", None, None]:
+        """
+        Split a current `ImageClassificationInput` object with a batch size b, into a
+        generator of b smaller objects with batch size 1, the returned
+        object can be iterated on.
+
+        :return: A Generator of smaller `ImageClassificationInput` objects each
+            representing an input of batch-size 1
+        """
+
+        images = self.images
+
+        # case 1: do nothing if single input of batch_size 1
+        if isinstance(images, str):
+            yield self
+
+        elif isinstance(images, list) and len(images) and isinstance(images[0], str):
```

Comment:
why is this only supporting string inputs?

Reply:
Changed to accept everything specified by Input Schemas